pandas/tests/io/formats/test_style.py

```diff
@@ -1682,6 +1682,26 @@ def f(a, b, styler):
         result = styler.pipe((f, "styler"), a=1, b=2)
         assert result == (1, 2, styler)
 
+    def test_chaining_table_styles(self):
+        df = pd.DataFrame(data=[[0, 1], [1, 2]], columns=["A", "B"])
```

Comment:
can you add the issue number (or PR number as i don't think there is an issue) on tests